Makefile

```diff
@@ -82,7 +82,7 @@ bin/$(ARCH)/$(BIN): build-dirs
 	    -v $$(pwd)/.go:/go                                                 \
 	    -v $$(pwd):/go/src/$(PKG)                                          \
 	    -v $$(pwd)/bin/$(ARCH):/go/bin                                     \
-	    -v $$(pwd)/bin/$(ARCH):/go/bin/$$(go env GOOS)_$(ARCH)             \
+	    -v $$(pwd)/bin/$(ARCH):/go/bin/linux_$(ARCH)                       \
```

Comment:
Hi @seh -- @thockin is doing this because of a quirk of how Go places binaries when you do a "go install".  It puts binaries for the local platform into `$GOROOT/bin` and the binaries for all cross targets in to `$GOROOT/bin/$GOOS/$ARCH`.  This requires a little bit of slight of hand to normalize everything.

Implicit in Tim's set up here, he is only building binaries for linux. There is no support for creating go binaries for either Mac or Windows (or anything else).  I'm guessing that this is because his ultimate goal is to create a container image and containers aren't well supported outside of Linux.  )Eventually I'd like to make this be OS aware so that we can create go binaries with this that aren't packaged in to containers, but that is for another time).

The problem here is that the `go env GOOS` is run on the "workstations" or "host" while the path being created is running inside of the machine running Docker.  If your workstation is running linux (like Tim's), this should work just fine.  But if your workstation is running something else (like macOS with Docker for Mac) then `go env GOOS` will resolve to `darwin`.  But we are trying to build linux binaries so this won't work!  This just admits that we are only building linux binaries and on a linux machine and hard codes that so that we can run from macOS.